site/code-snippets/web5/build/decentralized-web-nodes/ update-dwn.js

```diff
@@ -0,0 +1,9 @@
+export async function updateDWN
+const { record } = await web5.dwn.records.read({
+  message: {
+    recordId: 'bafyreiaz5oycqbrnmmpvffxqyoqxvx6bcnqueprmt2qnvzcurpc52r5uyy',
```

Comment:
```suggestion
    recordId: createdRecord.id,
```

This code will execute in real time. So, this record ID won't be valid. It was fine when it was just a non-executable example, but now we'll need a variable. This is why we passed in `createdRecord` as a function argument. Now we can use its id here.